stdlib/2/UserString.pyi

```diff
@@ -13,9 +13,9 @@ class UserString(Sequence[UserString]):
     def __hash__(self) -> int: ...
     def __len__(self) -> int: ...
     @overload
-    def __getitem__(self: _UST, i: int) -> _UST: ...
+    def __getitem__(self, i: int) -> UserString: ...
```

Comment:
I think the previous code here (and for the other overload) was correct. You can `# type: ignore` it if necessary.

Reply:
Reverting my change I get

```
stdlib/2/UserString.pyi:15: error: Signature of "__getitem__" incompatible with supertype "Sequence"
```